components/navigation/ui/src/main/resources/PhenoTips/DatabaseFilters.xml

```diff
@@ -61,6 +61,7 @@
 #if ($activeFields.contains('external_id'))
 * By **external identifier**: {{html clean=false}}&lt;p class="xHint"&gt;(The hospital ID manually entered)&lt;/p&gt;&lt;input type="text" name="external_id" value=""/&gt;{{/html}}
 #end
+* By **privacy level** {{html clean=false}}&lt;br&gt; #foreach ($visibility in $services.permissions.listVisibilityOptions()) &lt;input type="checkbox" class="multipleSelection privacyExport" name="visibility" value="$visibility" #if ($visibility != 'hidden')checked#end/&gt;$visibility&lt;br&gt;#end&lt;input type="hidden" name="visibility/class" value="PhenoTips.VisibilityClass"/&gt;{{/html}}
```

Comment:
`$visibility` is an instance of [`Visibility`](https://github.com/phenotips/phenotips/blob/master/components/patient-access-rules/api/src/main/java/org/phenotips/data/permissions/Visibility.java), not a `String`, so you're relying on the `toString` implementation to get the right string printed. That's not safe.

Internally, `toString()` just calls `getName()`, which is an internal identifier for the visibility option, which isn't translatable. Use `$visibility.name` for the value, `$visibility.label` for the label.